pyexchange/uniswapv2.py

```diff
@@ -119,6 +119,18 @@ def get_exchange_rate(self) -> Wad:
         else:
             return token_b_reserve / token_a_reserve
 
+    # retrieve exchange rate for the instance's pair token at a specified block
+    def get_exchange_rate_at_block(self, block_number: int) -> Wad:
+        assert (isinstance(block_number, int))
+
+        token_a_reserve = self.get_exchange_balance_at_block(self.token_a, self.pair_address, block_number)
+        token_b_reserve = self.get_exchange_balance_at_block(self.token_b, self.pair_address, block_number)
+
+        if token_a_reserve == Wad.from_number(0) or token_b_reserve == Wad.from_number(0):
```

Comment:
I don't think its actually possible to have one side of the exchange have zero tokens... is it?

Reply:
Regardless, I see value preventing the _divide by 0_ condition.